ref/addons.md

```diff
@@ -25,6 +25,7 @@ The following table lists the valid keys in an addon manifest:
 | `description`         | string       | Description of the addon.                                                                                                                                                                                                     |                                       |
 | `provides`            | object       | Additional capabilities that this addon provides to the container. See individual items for details.                                                                                                                          |                                       |
 | `provides.abi-support`| string array | Tells AMS that this addon adds support for the given architecture even if the application doesn't support it natively. Use this when your addon brings instruction translation or provides libraries for other architectures. | `arm64-v8a`, `armeabi-v7a`, `armeabi` |
+| `hooks.timeout`| string | Execution timeout for each hook included in an addon. The timeout is set by default to 5 minutes and can be extended up to 15 minutes. Configure this if a hook takes longer than 5 minutes to finish. | `10m` |
```

Comment:
```suggestion
| `hooks.timeout`| string | Execution timeout for each hook that is included in an addon. By default, the timeout is set to 5 minutes. It can be extended to up to 15 minutes. Configure this option if a hook takes longer than 5 minutes to finish. | `10m` |
```